pgmpy/estimators/StructureScore.py

```diff
@@ -88,62 +88,100 @@ def get_scoring_method(
 
 class StructureScore(BaseEstimator):
     """
-    Abstract base class for structure scoring classes in pgmpy. Use any of the
-    derived classes K2, BDeu, BIC or AIC. Scoring classes
-    are used to measure how well a model is able to describe the given data
-    set.
+    Abstract base class for structure scoring in pgmpy.
+
+    Structure scores are used to evaluate how well a given Bayesian network structure
+    fits observed data. This class should not be used directly. Use one of the derived
+    classes such as `K2`, `BDeu`, `BIC`, or `AIC` for concrete scoring methods.
+
+    Structure scores are central to model selection in Bayesian networks and are
+    particularly useful when comparing candidate network structures in discrete data scenarios.
```

Comment:
"discrete data scenarios" <- This is not true, scoring works for all data types.